.settings/org.eclipse.wst.common.component

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?><project-modules id="moduleCoreId" project-version="1.5.0">
```

Comment:
might need to update this branch. This file should be updated in master with these changes already